src/components/FeatureAvailability/FeatureAvailability.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../vars';
+
+.feature-availability {
+    border-radius: 4px;
+    padding: 8px 16px;
+    border: 1px solid $border;
+    margin-bottom: 16px;
+
+    h4 {
+        margin-top: 0;
+        font-size: 1em;
+    }
+
+    h5 {
+        margin: 0;
+        font-size: 0.9em;
+        text-transform: uppercase;
+        color: $text_muted_alt;
```

Comment:
This color looks a bit-off mark, I'd suggest just using the same color as headings or text for a more unified look (it's already that way in dark mode).

<img width="61" alt="Screenshot 2021-12-15 at 16 57 24" src="https://user-images.githubusercontent.com/4550621/146220166-2c15f825-f765-47a7-adbc-4c025be88802.png">